package.json

```diff
@@ -87,14 +87,17 @@
     "tap": "^15.0.1",
     "tape": "^5.0.0",
     "through2": "^4.0.0",
+    "transport": "./test/fixtures/transport",
```

Comment:
not a fan of adding this noise here, I know it's not a mistake, but it *looks* like a mistake. 

an alternative is to pass a `--loader` flag to the worker thread that points to test/fixtures/loader.mjs which would then resolve `transport` to /test/fixtures/transport.js within the worker thread

Reply:
I prefer not to rely on experimental features.  This way seemed the most "black box" testing that I could imagine.
The previous solution of `spawnSync` did not work on Windows.

Reply:
can we at least modify the package.json dynamically rather than check this in?

Reply:
I'll take this action

Reply:
Great!

Reply:
looks like it works without needing this in `package.json` at all

Reply:
scratch that, I'm an idiot